plugins/components/plugin-area/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { map } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import PluginRegistry from '../../api/plugin';
+import PluginContextProvider from '../plugin-context-provider';
+
+/**
+ * A component that renders all plugin fills in a hidden div.
+ */
+class PluginArea extends Component {
+	render() {
+		const pluginRegistry = PluginRegistry.getInstance();
+
+		return (
+			<div id="plugin-fills" style={ { display: 'none' } }>
+				{ map( pluginRegistry.plugins, plugin => {
```

Comment:
Should we be accessing properties of this singleton, or should this be a method from the API?